src/histogram.rs

```diff
@@ -166,10 +166,10 @@ pub struct HistogramCore {
     label_pairs: Vec<proto::LabelPair>,
 
     sum: AtomicF64,
-    count: AtomicU64,
+    count: AtomicI64,
```

Comment:
Why i64?

Reply:
Because AtomicU64 no longer exists, to support negative values in gauge. (Notice that gauge and counter both use Value so that 1, Value should support both of them 2, They have the same value domain: up to max i64 instead of u64) 

The AtomicI64 here is used for keeping the same value domain (I.e. max i64) as normal counters.